python/tests/test_preequilibration.py

```diff
@@ -234,3 +235,50 @@ def test_parameter_in_expdata(preeq_fixture):
             rdata_edata[variable][0, :],
             1e-6, 1e-6
         ).all(), variable
+
+
+def test_raise_postequilibration_with_adjoints(preeq_fixture):
+    """Test data replicates"""
+
+    model, solver, edata, edata_preeq, \
+        edata_presim, edata_sim, pscales, plists = preeq_fixture
+
+    # this needs to fail unless we remove presimulation
+    solver.setSensitivityMethod(amici.SensitivityMethod.adjoint)
+
+    rdata = amici.runAmiciSimulation(model, solver, edata)
+    assert rdata['status'] == amici.AMICI_ERROR
+
+    edata.t_presim = 0.0
+    edata.fixedParametersPresimulation = ()
+
+    rdatas = {}
+    for sensi_meth in [amici.SensitivityMethod.forward,
+                       amici.SensitivityMethod.adjoint]:
```

Comment:
That's just a question, not necessary, but:
Do we also want to not only check FSA and ASA, but also FD?